### ISSUE TITLE
Alt image text

### ISSUE THREAD

**Issue by ssh6** [CU-CommunityApps/CD_cwd_events_localist_pull#8]
It would be good to map the alt image tag for the pictures being returned to something...

**Comment by alisonjo315** [comment]
Yeah it's weird to have it left empty by default, and it's required.

@ssh6 Is there an "alt" field in localist -- when you add the image to the event in Cornell Calendar, do you enter alt text, or no?  I don't see a field for it in the JSON ([example](https://events.cornell.edu/api/2/events?api_key=KLhy2GtuSAGirYGY)).

Actually, I might recommend disabling the alt text field, because we don't recommend using it anyway, we recommend an empty alt attribute for listing pages ([like this](https://einaudi.cornell.edu/engage/events)).

**Comment by ssh6** [comment]
Looking at the events feed... there is nothing in the feed for alt tags.  

`id                : 37077792089348                                                                                                  
title             :  Clothing Amidst Conflict: Womenswear During World War II                                                       
url               :                                                                                                                 
updated_at        : 7/13/2021 8:59:02 AM
created_at        : 6/22/2021 3:56:53 PM
facebook_id       : 
first_date        : 2021-06-23
last_date         : 2021-08-13
hashtag           : 
urlname           : clothing_amidst_conflict_womenswear_during_world_war_ii
user_id           : 8793
directions        : 
allows_reviews    : True
allows_attendance : True
location          : 
room_number       : Terrace Level Display Cases
location_name     : Human Ecology Building (HEB)
status            : live
experience        : inperson
stream_url        : 
stream_info       : 
stream_embed_code : 
created_by        : 8793
updated_by        : 37103016146763
city_id           : 
neighborhood_id   : 
school_id         : 10
campus_id         : 
recurring         : True
free              : False
private           : False
verified          : True
rejected          : False
sponsored         : False
venue_id          : 70448
ticket_url        : 
ticket_cost       : 
keywords          : {}
tags              : {}
description_text  : A new fashion exhibition, Clothing Amidst Conflict: Womenswear During World War II, will feature clothing worn 
                    by American servicewomen, medical professionals, and civilians from 1941 to 1945. The exhibit will be in The 
                    College of Human Ecology Terrace Level Display Cases through August 15, 2021.
photo_id          : 37077792323888
detail_views      : 135
event_instances   : {@{event_instance=}}
address           : 42.4505,-76.4786
description       : <p>A new fashion exhibition, Clothing Amidst Conflict: Womenswear During World War II, will feature clothing 
                    worn by American servicewomen, medical professionals, and civilians from 1941 to 1945. The exhibit will be in 
                    The College of Human Ecology Terrace Level Display Cases through August 15, 2021.</p>
                    
featured          : False
geo               : @{latitude=42.450478; longitude=-76.478523; street=37 Forest Home Drive; city=Ithaca; state=NY; country=US; 
                    zip=14850}
filters           : @{departments=System.Object[]; event_types=System.Object[]}
custom_fields     : @{contact_email=dng22@cornell.edu; contact_name=Denise Green}
localist_url      : https://events.cornell.edu/event/clothing_amidst_conflict_womenswear_during_world_war_ii
localist_ics_url  : https://events.cornell.edu/event/clothing_amidst_conflict_womenswear_during_world_war_ii.ics
photo_url         : https://localist-images.azureedge.net/photos/37077792323888/huge/a71deefe53bd8e1dfd587a257fbb0c384643aa46.jpg
venue_url         : https://events.cornell.edu/human_ecology_building_heb_486

`
But looking at the event in events.cornell.edu ( https://events.cornell.edu/event/clothing_amidst_conflict_womenswear_during_world_war_ii)

The image is <img src="https://localist-images.azureedge.net/photos/37077792323888/original/a71deefe53bd8e1dfd587a257fbb0c384643aa46.png" alt="">

But if you see it in search results, you see:<img alt=" Clothing Amidst Conflict: Womenswear During World War II" class="img_big" height="150" src="https://localist-images.azureedge.net/photos/37077792323888/big/a71deefe53bd8e1dfd587a257fbb0c384643aa46.jpg" srcset="https://localist-images.azureedge.net/photos/37077792323888/big@2x/a71deefe53bd8e1dfd587a257fbb0c384643aa46.jpg 2x, https://localist-images.azureedge.net/photos/37077792323888/big/a71deefe53bd8e1dfd587a257fbb0c384643aa46.jpg 1x" width="200">

So it looks like we will need to tackle localist/localist feed before we worry about the module.

**Comment by alisonjo315** [comment]
Thanks @ssh6!

I think for most (maybe all) of our use cases, we don't need the alt text anyway -- we render event lists with the image and title inside one `<a>` tag, so the alt attribute is supposed to be present but empty for ideal web accessibility.  With that in mind, I'm recommending that for our sites, we disable alt text on `field_event_image`.  This will be a change to our Drupal demo site/starter-kit/install profile thing, not the localist importer module.

For now, I'll add "enhancement" and "discussion" labels to this issue.
